ocode_python/core/orchestrator.py

```
@@ -0,0 +1,571 @@
+                priority=priority,
+            )
+            task_objects.append(task)
+
+        if parallel:
+            return await self.concurrent_executor.execute_parallel_tools(
+                task_objects, self.tool_registry, self.side_effect_broker
+            )
+        else:
+            results = []
+            for task in task_objects:
+                result = await self.concurrent_executor._execute_single_task(
```

Pylint (3.3.7)

[warning] Access to a protected member _execute_single_task of a client class

(W0212)